pkg/bitwarden/cli.go

```diff
@@ -0,0 +1,153 @@
+package bitwarden
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+)
+
+type bitwardenCliClient struct {
```

Comment:
This type stutters, too. `bitwarden.cliClient` is fine

Reply:
Done.